packages/clarity-js/src/data/consent.ts

```diff
@@ -1,5 +1,16 @@
-import { Dimension } from "@clarity-types/data";
+import { Dimension, Event } from "@clarity-types/data";
 import * as dimension from "@src/data/dimension";
+import encode from "./encode";
+
+export let data: string = "";
```

Comment:
this should be a consent object not a json string - and encode different fields of the object